README.md

```diff
@@ -4,3 +4,17 @@
 
 
 A Flask based web service used by the public to get access to Civil Legal Aid services.
+
+# MOJ Flask Skeleton
+
+The MOJ Flask Skeleton is set as a subtree from https://github.com/ministryofjustice/moj-flask-govuk-skeleton and is kept in the laa_access_civil_legal_aid folder.
```

Comment:
Is it worth adding a documentation link to "subtree" so people can easily find out what that means?